content/zh-cn/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -14,6 +15,8 @@ reviewers:
 - shu-mutou
 - mikedanese
 title: Deploy and Access the Kubernetes Dashboard
+description: >-
+  Deploy the web UI (Kubernetes Dashboard) and access it.
 content_type: concept
 weight: 10
```

Comment:
```suggestion
weight: 30
```

Reply:
What is the reason for such a change？Emm...

Reply:
Your title is to `sync` but the current zh has a different weight from the commented weight.

Reply:
EN section as follows:
```md
reviewers:
- floreks
- maciaszczykm
- shu-mutou
- mikedanese
title: Deploy and Access the Kubernetes Dashboard
description: >-
  Deploy the web UI (Kubernetes Dashboard) and access it.
content_type: concept
weight: 10
card:
  name: tasks
  weight: 30
  title: Use the Web UI Dashboard
```

CN section as follows:
```md
title: 部署和访问 Kubernetes 仪表板（Dashboard）
content_type: concept
weight: 10
card:
  name: tasks
  weight: 30
  title: 使用 Web 界面 Dashboard
  description: 部署并访问 Web 界面（Kubernetes 仪表板）。
```

I am sorry, do I miss something? Or  you missed up `weight` with `card.weight`?

Reply:
oh oh, sorry, you are correct! -_!